commit 32d597f993f2566f7bdfec8a29c916b4c9b1a803
Author: Darrell Anderson <humanreadable@yahoo.com>
Date:   1339639807 -0500

    Reverse an old Debian-specific patch that prevents building on
    some 64-bit systems. Add text to help with basic debugging.

diff --git a/setup.py b/setup.py
index e51f0fa..e7ec147 100755
--- a/setup.py
+++ b/setup.py
@@ -135,7 +135,7 @@ class BuildLibpythonize(Command):
                     if self.clib!=None:
                         self.clib = glob.glob(os.path.join(self.clib,'libgcc*.a'))[0]
             if self.clib is None:
-                raise SystemExit, "Failed to find a suitable libgcc library"
+                raise SystemExit, "setup.py: Failed to find a suitable libgcc library"
             self.announce("Using %s for clib" % self.clib)
     
             # Make a list of places to look for python .so modules
diff --git a/src/kdedistutils.py b/src/kdedistutils.py
index b8b3743..b8e23f3 100644
--- a/src/kdedistutils.py
+++ b/src/kdedistutils.py
@@ -660,7 +660,7 @@ class BuildKControlModule(Command):
                     if self.clib!=None:
                         self.clib = glob.glob(os.path.join(self.clib,'libgcc*.a'))[0]
             if self.clib is None:
-                raise SystemExit, "Failed to find a suitable libgcc library"
+                raise SystemExit, "kdedistutils.py (1): Failed to find a suitable libgcc library"
             self.announce("Using %s for clib" % self.clib)
     
             # Make a list of places to look for python .so modules
@@ -1591,7 +1591,7 @@ class BuildKioslave(Command):
                     if self.clib!=None:
                         self.clib = glob.glob(os.path.join(self.clib,'libgcc*.a'))[0]
             if self.clib is None:
-                raise SystemExit, "Failed to find a suitable libgcc library"
+                raise SystemExit, "kdedistutils.py (2): Failed to find a suitable libgcc library"
             self.announce("Using %s for clib" % self.clib)
     
             # Make a list of places to look for python .so modules
@@ -2229,7 +2229,7 @@ def compile_qtdesigner(ui_files,
 
 ###########################################################################
 def get_libdir_name():
-    #if os.uname()[4] in ['x86_64','mips64','ppc64','sparc64','s390x']:
-    #    return 'lib64'
-    #else:
+    if os.uname()[4] in ['x86_64','mips64','ppc64','sparc64','s390x']:
+        return 'lib64'
+    else:
         return 'lib'
